guibot/finder.py

```diff
@@ -32,11 +32,14 @@
 import random
 import configparser as config
 import PIL.Image
+from typing import Callable
 
 from .config import GlobalConfig, LocalConfig
 from .imagelogger import ImageLogger
 from .fileresolver import FileResolver
 from .errors import *
+from .match import Match
```

Comment:
This should be annotated via quotation marks.

Reply:
Done